src/plugins/conditionals/README.md

```diff
@@ -103,20 +103,19 @@ sudo kdb mount main.ini /examples/conditionals ni
 sudo kdb mount sub.ini /examples/conditionals/sub ini
 
 # mount conditionals as global plugin
-sudo kdb global-mount conditionals
+sudo kdb global-mount conditionals || $(exit 0)
```

Comment:
Did it fail for you? I would prefer to have an explicit test for the assumption that might be broken on your system (e.g. conditionals not available?) instead of silently stop the test.

With your proposed change `kdb global-mount` could be broken and we would not notice.